packages/components/psammead-episode-list/src/fixtures.jsx

```diff
@@ -43,75 +55,201 @@ export const exampleEpisodes = [
+    margin-bottom: ${GEL_SPACING_DBL};
+  }
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_TRPL};
+  }
+  ${({ darkMode }) => darkMode && `color: ${C_WHITE}`}
+`;
+
+const SurroundingComponents = ({
+  children,
+  script,
+  service,
+  dir,
+  darkMode,
+}) => (
+  <Spacer darkMode={darkMode}>
+    <StyledSectionLabel
+      script={script}
+      service={service}
+      dir={dir}
+      darkMode={darkMode}
+      {...(darkMode ? { backgroundColor: C_MIDNIGHT_BLACK } : {})}
+    >
+      Recent Episodes
+    </StyledSectionLabel>
+    {children}
+  </Spacer>
+);
+
+export const renderEpisodes = ({
+  episodes,
+  script,
+  service,
+  dir,
+  withSurroundingComponents,
+  darkMode,
+}) => {
+  const Wrapper = withSurroundingComponents
+    ? SurroundingComponents
+    : React.Fragment;
+  return (
+    <Wrapper
+      {...(withSurroundingComponents ? { script, service, dir, darkMode } : {})}
+    >
+      <EpisodeList script={script} service={service} dir={dir}>
+        {episodes.map(episode => (
+          <EpisodeList.Link key={episode.id} href={episode.url}>
+            <EpisodeList.Episode>
+              {/* eslint-disable-next-line jsx-a11y/aria-role */}
+              <span role="text">
+                <VisuallyHiddenText>Audio, </VisuallyHiddenText>
+                <EpisodeList.Title className="underlined_hover fade_visited">
+                  {episode.brandTitle}
+                </EpisodeList.Title>
+                <VisuallyHiddenText>, </VisuallyHiddenText>
+                <EpisodeList.Description className="underlined_hover fade_visited">
+                  {episode.episodeTitle || `${episode.date}, ${episode.time}`}
+                </EpisodeList.Description>
+                <VisuallyHiddenText>, </VisuallyHiddenText>
+                <VisuallyHiddenText>
+                  {` ${episode.durationLabel} ${formatDuration({
+                    duration: episode.duration,
+                    format: episode.duration.includes('H')
+                      ? 'h,mm,ss'
+                      : 'mm,ss',
+                    locale: episode.locale,
+                  })} `}
+                </VisuallyHiddenText>
+                <EpisodeList.Metadata>
+                  <span aria-hidden="true">
+                    {` ${episode.durationLabel} ${formatDuration({
+                      duration: episode.duration,
+                      locale: episode.locale,
+                    })}`}
+                  </span>
+                  {episode.episodeTitle && (
+                    <span aria-hidden>
+                      {' '}
+                      <StyledSpan>|</StyledSpan> {episode.date}
+                    </span>
+                  )}
+                </EpisodeList.Metadata>
+              </span>
+            </EpisodeList.Episode>
+          </EpisodeList.Link>
+        ))}
+      </EpisodeList>
+    </Wrapper>
+  );
+};
+
+export const renderVideoEpisodes = ({
```

Comment:
Need to provide an example implementation of the a11y requirements here

May also negate this function when done - could end up being very similar to the function for rendering audio episodes